identity-service/src/routes/twitter.js

```diff
@@ -128,6 +129,9 @@ module.exports = function (app) {
           const contractRegKey = await audiusLibsInstance.contracts.getRegistryContractForAddress(contractAddress)
           const senderAddress = config.get('userVerifierPublicKey')
 
+          // Ensure social handle matches Twitter handle for verified accts.
+          handle = twitterObj.twitterProfile.screen_name
```

Comment:
@SidSethi i'm not sure this does what we want. this sets the audius handle in the SocialHandles table effectively serving as the primary key. I actually think this is pretty much what we want for the associate call

Reply:
reverted